.github/workflows/code-cover-publish.yaml

```diff
@@ -0,0 +1,58 @@
+name: PR code coverage (publish)
+
+on:
+  workflow_run:
+    workflows: [ "PR code coverage (generate)" ]
+    types: [ "completed" ]
+
+
+jobs:
+  # This job downloads the artifacts generated by the code-cover-gen job and
+  # uploads them to a GCS bucket, from where Reviewable can access them.
+  #
+  # Note that this workflow is not required for a PR to merge; a failure simply
+  # means that there won't be coverage data visible in Reviewable.
+  code-cover-publish:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
```

Comment:
So this step will run even if the `generate` step was skipped? Won't `matchArtifact` find no matches (or even crash since it's referencing `[0]` directly).